source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/stores/-components/form.tsx

```diff
@@ -3,7 +3,7 @@ import { withValue } from 'react-state-helpers';
 
 import { query } from '@data';
 
```

Comment:
this file is just converted to hooks, which uses the `useQuery` hook from dt/react-orbitjs, which has some race-condition fixes over the HoC version of query